### DISCUSSION TITLE
Test installs for webui download via torrent link.

### DISCUSSION THREAD

**Discussion by wmealing** [rhinstaller/anaconda-webui#44] [General]
The current latest install  link ( https://fedorapeople.org/groups/anaconda/webui_preview_image/x86_64/webui_latest_install.iso ) is downloading, but it is very slow, please make future tests available via a torrent.  

This is not a priority, it is only nice to have.

Thanks.

Thank you !

**Comment by jkonecny12** [comment]
Hi @wmealing sounds like a good idea, however, not sure if we found time for that. Anyway, we are right now working on size reduction of the ISO so the situation should be better.

**Reply by jkonecny12** [reply]
**DISCLAIMER**: the below links are not ours and are not reliable for them.

-------------------------

Someone from the community already created a torrent and I've added a sha256 sums to the files for verification.

```
magnet:?xt=urn:btih:fadc50581815a88d2d8386f9ce3f43e1de387969&dn=fedora-preview-installer-2022-11-23&tr=udp%3a%2f%2ftracker.opentrackr.org%3a1337%2fannounce&tr=udp%3a%2f%2fopen.stealth.si%3a80%2fannounce
```

with SHA for confirmation of the ISO:
https://fedorapeople.org/groups/anaconda/webui_preview_image/x86_64/fedora-preview-installer-2022-11-23.iso-SHA256CHECKSUM

**Reply by wmealing** [reply]
Thanks!

**Comment by wmealing** [comment]
Thank you, you can consider this closed!
